### PR TITLE
README.md changes

### PR DESCRIPTION
**Pull request by hm90121**
--added table of contents
-- zus overview
-- added sample migration command syntax
-- added build instructions
-- corrected typos

### CODE REVIEW

**Review by boddumanohar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,38 +1,66 @@
-# s3-migration - a CLI for migrating AWS s3 bucket to 0Chain dStorage 
-This program helps to migrate files in s3 bucket to 0chain dStorage.
+# s3-migration - a CLI for migrating AWS S3 bucket to Züs dStorage
```
Comment (Kishan-Dhakan) [inline README.md]: `a CLI tool` please

File: README.md [hunk 2]
```diff
@@ -1,38 +1,66 @@
...
 [migrate](#migrate)|Migrate user data from S3 bucket to dStorage
 
-## Running Test cases
-
-`go test ./...`
-
 ### Global Flags
 
-Global Flags are parameters in s3mgrt that can be used with any command to override the default configuration supports the following global parameters.
+Global Flags are parameters in s3igration that can be used with any command to override the default configuration supports the following global parameters.
```
Comment (Kishan-Dhakan) [inline README.md]: s3migration (spelling)

File: README.md [hunk 3]
```diff
@@ -1,38 +1,66 @@
...
+| --wallet_client_key string | Specify a wallet client_key (By default client_key specified in $HOME/.zcn/wallet.json is used) | ./s3migration migrate --wallet_client_key  "client key" |
+| --wallet_private_key string | Specify wallet private key| ./s3migration migrate --wallet_private_key "wallet private key"|
 
 ## Migrate Command
 
-`migrate` command is used to migrate files from s3 buckets to some remote directory(default is /) by using aws-s3-sdk and 0chain gosdk. All the objects from bucket will be migrated.
-However user can specify some prefix to migrate only the files with those prefix. Also if there is name conflict within dStorage file and bucket file use can
+`migrate` command is used to migrate files from s3 buckets to some remote directory(default is /) by using aws-s3-sdk and 0chain gosdk. All the objects from the bucket will be migrated.
+However users can specify some prefix to migrate only the files with those prefix. Also if there is name conflict within dStorage file and bucket file use can
```
Comment (Kishan-Dhakan) [inline README.md]: Please change to `dStorage file and bucket file users can` (typo in users)

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Comment by hm90121** [comment]
applied your suggestions

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**